<launch>
  <include file="$(find rm_description)/launch/hero.launch">
    <arg name="load_chassis" value="true"/>
    <arg name="roller_type" value="realistic"/>
    <arg name="paused" value="true"/>
  </include>

  <rosparam file="$(find hero_chassis_controller)/cfg/controllers.yaml" command="load"/>

  <!-- launch dynamic reconfigure node -->
  <node name="dynamic_reconfigure" pkg="hero_chassis_controller" type="dynamic_reconfigure_node" respawn="false" output="screen"/>

  <!-- cmd_vel_mux -->
  <node name="cmd_vel" pkg="hero_chassis_controller" respawn="false" type="cmd_vel" output="screen"/>

<!--  &lt;!&ndash; teleop_twist_keyboard &ndash;&gt;-->
<!--  <node name="teleop_twist_keyboard" pkg="teleop_twist_keyboard" type="teleop_twist_keyboard.py" output="screen"/>-->

  <!-- boost controller -->
  <node name="controller_spawner" pkg="controller_manager" type="spawner" respawn="false"
        output="screen" args="
        controller/hero_chassis_controller
        controller/joint_state_controller
  "/>
</launch>